lib/utils.js

```diff
@@ -232,5 +247,6 @@ module.exports = {
   endOfMonthTimestamp,
   calculateQuotaExpiration,
   resolveElevatedParams,
-  replicateHashtag
+  replicateHashtag,
+  isFixedWindowEnabled
```

Comment:
You can put trailing commas on exports, which simplifies diffs when new stuff needs to be added

```suggestion
  isFixedWindowEnabled,
```

Reply:
fixed